Src/tim.c

```diff
@@ -53,9 +53,13 @@ void MX_TIM16_Init(void)
   TIM_BreakDeadTimeConfigTypeDef sBreakDeadTimeConfig;
 
   htim16.Instance = TIM16;
-  htim16.Init.Prescaler = 0;
+
+  // 16bit timer is 65535 counts. Desired period is 20ms. Clock frequency is 48MHz
+  // With a prescaler of 32 the period would be 65535/(48M/32) = 43.69ms.
+  // With the period being set to (65535)/2, the period would be 43.69/2 = 21.845ms
+  htim16.Init.Prescaler = 16;
```

Comment:
Are you using a prescalar of 16 or 32? Just asking because there is a diff between your comments and code